apps/rehabstod/src/components/Table/TablePageLayout.tsx

```diff
@@ -27,10 +30,23 @@ export function TablePageLayout({
   printable: boolean
   children: ReactNode
 }) {
+  const { data: doctorsFilterResponse } = useGetDoctorsForLUCertificatesQuery()
+  const { data: user } = useGetUserQuery()
+  const isDoctor = user ? isUserDoctor(user) : false
+
+  function noDoctorsForLU() {
+    return doctorsFilterResponse && doctorsFilterResponse?.doctors.length === 0
+  }
+
   return (
     <PageContainer>
       {heading}
       {filters}
+      {noDoctorsForLU() && (
+        <IDSAlert className="py-10">
+          {isDoctor ? 'Du har' : 'Det finns'} inga läkarutlåtanden på {user && user.valdVardenhet ? user.valdVardenhet.namn : ''}.
```

Comment:
This component is generic so we shouldn't add anything related to sickLeave / LU here. I think it would be nice to add this as a EmptyTableAlert component and send in the tableName and user instead. Similar to how it's done in TableInfoMessage and TableContentAlert

Reply:
Good point, I will correct this!

Reply:
Updated PR!